azurerm/resource_arm_app_service.go

```diff
@@ -388,6 +388,7 @@ func resourceArmAppServiceRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("client_affinity_enabled", props.ClientAffinityEnabled)
 		d.Set("enabled", props.Enabled)
 		d.Set("default_site_hostname", props.DefaultHostName)
+		d.Set("outbound_ip_addresses", props.OutboundIPAddresses)
```

Comment:
can we add a matching Schema field for this (otherwise this won't actually set anything) - there's [an example of this here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_app_service.go#L233), and it'll need to be Computed like the `default_site_hostname` field since it's only set once the app is created

Reply:
done